index.js

```diff
@@ -89,7 +89,7 @@ var Sqlite3Driver = Base.extend({
     if (spec.defaultValue !== undefined) {
       constraint.push('DEFAULT');
 
-      if(typeof(spec.defaultValue) === 'string')
+      if(typeof(spec.defaultValue) === 'string' && spec.defaultValue.indexOf('(') === -1)
```

Comment:
This would break every defaultValue that is a string containing `(`, therefore this is not a safe way to check for a function call.